apps/planty/src/app/solid-core-config.ts

```diff
@@ -10,4 +10,5 @@ export const coreConfig: SolidCoreConfig = {
   error_report: true,
   guidedTour: guidedTour.landing,
   profileTour: guidedTour.profile,
+  profileLocation: JSON.stringify(guidedTour.location[0].url),
```

Comment:
In diesem Namen (`profileLocation`) sollte noch irgendwo "Tour" stehen, damit eindeutig ist, wozu es gehört.